Reply/app/src/main/java/com/materialstudies/reply/ui/common/SearchSuggestionItem.kt

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.materialstudies.reply.ui.common
+
+import android.view.ViewGroup
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.focusable
+import androidx.compose.foundation.horizontalScroll
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import com.google.accompanist.themeadapter.material.MdcTheme
+import com.materialstudies.reply.R
+import com.materialstudies.reply.data.SearchSuggestion
+import com.materialstudies.reply.data.SearchSuggestionStore
+import com.materialstudies.reply.ui.theme.workSansFontFamily
+import com.materialstudies.reply.util.ThemePreviews
+
+@Composable
+fun SearchSuggestionItem(
+    searchSuggestion: SearchSuggestion,
+    modifier: Modifier = Modifier
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = modifier
+            .clickable {}
```

Comment:
Should this function expose an `onClick`?

Reply:
This feature was never originally implemented, the click doesn't do anything. I added the clickable modifier here only for the visual, ripple effect

Reply:
Gotcha. Maybe just add a comment why it's empty?